packages/code-block/mdx/index.js

```diff
@@ -26,7 +26,7 @@ export function pre({
   }
   // Assert that the one valid child is a <code> element
   const codeChild = childArray[0]
-  if (codeChild.props.mdxType !== 'code') {
+  if ((codeChild.props.mdxType || codeChild.type) !== 'code') {
```

Comment:
`mdxType` is no longer a prop that's passed, so we check the component type itself.